Core/Services/StitchCoreRemoteMongoDBService/Sources/StitchCoreRemoteMongoDBService/Sync/Internal/DocumentVersionInfo.swift

```diff
@@ -64,7 +64,7 @@ final class DocumentVersionInfo {
         /// the GUID instance id of this version
         let instanceId: String
         /// the version counter of this version
-        let versionCounter: Int64
+        let versionCounter: Int
     }
```

Comment:
Can you explain this? Why do we no longer have to specify it as a long?

Reply:
It makes it easier to work with when testing if it's an `Int`. Plus:
https://stackoverflow.com/questions/27440100/what-is-the-difference-between-int-and-int32-in-swift

Reply:
Shouldn't this be consistent between implementations? -- in Java it's a long which is 64 bits on all architectures

Reply:
I've changed it back

Reply:
I mean, ostensibly it shouldn't matter since a swift vs java client would have different instance IDs and the likelihood of us maxing that out are nil... but still